docs/web-console-guide_5/master.adoc

```diff
@@ -57,10 +57,70 @@ include::topics_5/web-analyze-apps.adoc[leveloffset=+1]
 include::topics_5/web-add-project.adoc[leveloffset=+2]
 
 // Add Applications to a Project
-include::topics_5/web-add-applications.adoc[leveloffset=+2]
+// include::topics_5/web-add-applications.adoc[leveloffset=+2]
+
+=== Adding applications to a project
+
+A project must contain one or more applications.
+
+You can add applications to a project by:
+
+* Uploading applications
+* Registering the directory path of each application
+
+Uploading applications stores them directly in the {ProductShortName} server.
+
+Registering the directory path of an application tells the {ProductShortName} server the location of the application. This method is useful if you expect to update the application because it ensures that the {ProductShortName} server always uses the latest version of the application.
+
+.Adding applications
+image::web-add-apps_51.png[Adding applications]
+
```

Comment:
Uploading and Registering path are not separate procedures!!!! They are options for a single step in the same procedure!